### PR TITLE
Reenable prevcycle

### PR DESCRIPTION
**Pull request by matthiasgeiger**
Okay... this should be working again now. However, without the CSL styles working (see #5246) it's a bit hard to check. ;-) 

Fixes https://github.com/JabRef/jabref/issues/5221

----

- [x] Change in CHANGELOG.md described
- ~~[ ] Tests created for changes~~
- [x] Manually tested changed features in running JabRef
- ~~[ ] Screenshots added in PR description (for bigger UI changes)~~
- [x] Ensured that [the git commit message is a good one](https://github.com/joelparkerhenderson/git_commit_message)
- ~~[ ] Check documentation status (Issue created for outdated help page at [help.jabref.org](https://github.com/JabRef/help.jabref.org/issues)?)~~

### CODE REVIEW

File: src/main/java/org/jabref/gui/entryeditor/EntryEditor.java [hunk 1]
```diff
@@ -411,4 +411,20 @@ public void updatePreviewInTabs(PreviewPreferences previewPreferences) {
             }
         }
     }
+
+    public void nextPreviewStyle() {
+        for (Tab tab : this.entryEditorTabs) {
+            if (tab instanceof FieldsEditorTab) {
+                ((FieldsEditorTab) tab).previewPanel.nextPreviewStyle();
```
Comment (tobiasdiez) [inline src/main/java/org/jabref/gui/entryeditor/EntryEditor.java]: I would add `next/previousPreviewStyle` to `EntryEditorTab` and override it properly in `FieldsEditorTab` . In this way, we are a bit more flexible in the future (e.g. if there are other tabs with previews embedded) and you can make `previewPanel` private (at least it should be private).

File: src/main/java/org/jabref/gui/preview/PreviewPanel.java [hunk 2]
```diff
@@ -167,4 +154,20 @@ public void setEntry(BibEntry entry) {
...
+    public void previousPreviewStyle() {
+        cyclePreview(Globals.prefs.getPreviewPreferences().getPreviewCyclePosition() - 1);
+    }
+
+    private void cyclePreview(int newPosition) {
+        PreviewPreferences previewPreferences = Globals.prefs.getPreviewPreferences()
+                .getBuilder()
+                .withPreviewCyclePosition(newPosition)
+                .build();
+        updateLayout(previewPreferences);
```
Comment (tobiasdiez) [inline src/main/java/org/jabref/gui/preview/PreviewPanel.java]: Is there no need to store the new layout in the preferences? (Should probably be done in updateLayout)

File: src/main/java/org/jabref/gui/preview/PreviewPanel.java [hunk 3]
```diff
@@ -167,4 +154,20 @@ public void setEntry(BibEntry entry) {
     public void print() {
         previewView.print();
     }
+
+    public void nextPreviewStyle() {
+        cyclePreview(Globals.prefs.getPreviewPreferences().getPreviewCyclePosition() + 1);
```
Comment (tobiasdiez) [inline src/main/java/org/jabref/gui/preview/PreviewPanel.java]: Please extract `Globals.prefs` as a constructor argument (probably replacing `PreviewPreferences`) as we want to get rid of `Globals` as much as possible.

File: src/main/java/org/jabref/gui/preview/PreviewPanel.java (hunk 2 above)
Comment (matthiasgeiger) [reply src/main/java/org/jabref/gui/preview/PreviewPanel.java]: Yes, good catch!

File: src/main/java/org/jabref/gui/entryeditor/EntryEditor.java (hunk 1 above)
Comment (matthiasgeiger) [reply src/main/java/org/jabref/gui/entryeditor/EntryEditor.java]: Done.

**Pull request merged**